ext/lib/crypto/tinycrypt/include/tinycrypt/ccm_mode.h

```diff
@@ -202,7 +202,7 @@ int tc_ccm_generation_encryption(uint8_t *out, unsigned int olen,
 int tc_ccm_decryption_verification(uint8_t *out, unsigned int olen,
 				   const uint8_t *associated_data,
 				   unsigned int alen, const uint8_t *payload, unsigned int plen,
-				   TCCcmMode_t c);
+				   TCCcmMode_t c, const uint8_t *pTag);
 
 #ifdef __cplusplus
 }
```

Comment:
don't like the idea of breaking tinycrypt API just to make our life easier. This pattern of adding payload + tag/IV in a contiguous buffer is common in tinycrypt.

Reply:
I change tinycrypt API just as my concerns.